fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -229,7 +230,8 @@
     protected ExternalContentHandlerFactory extContentHandlerFactory;
 
     private static final Predicate<Triple> IS_MANAGED_TYPE = t -> t.getPredicate().equals(type.asNode()) &&
-            isManagedNamespace.test(t.getObject().getNameSpace());
+            (isManagedNamespace.test(t.getObject().getNameSpace()) ||
+            isManagedLDPType.test(createResource(t.getObject().getURI())));
```

Comment:
Just wondering if this should be removed and replaced with the `isManagedPredicate` from `RdfLexicon` which contains other stuff as well?
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java#L280-L281

Reply:
Also we shouldn't be suppressing `<> rdf:type ldp:BasicContainer` when we suppress SMTs. This is not an SMT, I think

Reply:
The "Are LDP Container types SMTs" question should be resolved on the 2018-10-04 Tech call.

As for the `isManagedPredicate`, the test here is:
- is the predicate `rdf:type`, and
- is the object in a managed namespace, or
- is the object a managed LDP type

@whikloj : are you suggesting the possibility of changing the logic to:
- is the predicate `rdf:type`, and
- is the predicate managed?

Reply:
I am just unclear as to what is an SMT and that makes testing for them very difficult. I think I should just wait until that is decided then I can have an opinion.

Reply:
So following the discussion on the tech call my opinion is that the existing `isManagedNamespace` predicate could be expand to include both `fedora:*` and `ldp:*` and then this change to add `isManagedType` would be unnecessary. Was that your understand as well @mohideen @awoods?

Reply:
@whikloj : That sounds right based on today's call.

Reply:
@whikloj @awoods So, we won't be able to create resources with `ldp:*` rdf:type?

```
$ curl -i -u fedoraAdmin:fedoraAdmin http://localhost:8080/rest/testputldp -X PUT -H "Content-Type: text/turtle" --data-binary '<> a <http://www.w3.org/ns/ldp#DirectContainer> .'
HTTP/1.1 409 Conflict
Date: Thu, 04 Oct 2018 23:44:06 GMT
Set-Cookie: JSESSIONID=186v9jsw7ckum1qnzffgc3lp6s;Path=/
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Set-Cookie: rememberMe=deleteMe; Path=/; Max-Age=0; Expires=Wed, 03-Oct-2018 23:44:06 GMT
Link: <http://localhost:8080/static/constraints/ServerManagedTypeException.rdf>; rel="http://www.w3.org/ns/ldp#constrainedBy"
Content-Type: text/plain;charset=utf-8
Content-Length: 98
Server: Jetty(9.3.1.v20150714)

The repository type (http://www.w3.org/ns/ldp#DirectContainer) of this resource is system managed.
```

Did we say we will use `Link` headers to overcome this?

Reply:
Refactored based on today's decisions.